### ISSUE TITLE
[CoE Starter Kit - FEATURE] Impact Analysis for disabling code components

### ISSUE THREAD

**Issue by manuelap-msft** [microsoft/coe-starter-kit#5024]
### What is your question?

From Champions:

Impact analsyis on what would happen if code components were to be disabled on an environment. 

### What solution are you experiencing the issue with?

None

**Comment by manuelap-msft** [comment]
Once #4744 is done, explore if we can figure out if an app is using code components through inventory.

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]:https://github.com/microsoft/coe-starter-kit/issues/10333

Thank you for your contributions
